keyBy.js

```diff
@@ -12,6 +12,7 @@ import reduce from './reduce.js';
  * @param {Array|Object} collection The collection to iterate over.
  * @param {Function} iteratee The iteratee to transform keys.
  * @returns {Object} Returns the composed aggregate object.
+ * @see [groupBy](#groupBy), [partition](#partition)
  * @example
```

Comment:
It can just be `@see groupBy, partition` no need to add markdown links as that's something for our doc generation to tackle.